src/app.ts

```diff
@@ -99,6 +99,13 @@ const editorActions = {
   loadTextNote: (nodeId: GraphNodeId) => async (state: any, actions: any) => {
     const data = await loadNote(nodeId, NoteDataType.TEXT)
     actions.textEditor.setData(data)
+
+    // Dispatch a custom event to update the default CodeMirror text.
+    const updateEvent = new CustomEvent('textupdate', {
+      detail: {data},
+    })
+    const textEditor = (document as any).getElementById('text-editor')
```

Comment:
We'll need to consider a different approach. I don't want to start relaying on `getElementById` arbitrarily. It causes issues with the mental model you develop since you're relying on hyperapp to manage state and UI. Accessing things like is subverting the existing system and will only lead to headaches down the line

Reply:
Fixed.